<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evolution v 0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="gameLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="setLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="controlLayout">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Universe Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cellsControl">
         <property name="suffix">
          <string> cells</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="cellsSlider">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Generation interval (in msec)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterInterval">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="iterSlider">
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="sliderPosition">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fileLayout">
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="statsLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="labelsBox">
           <property name="title">
            <string>Color</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="stAllLab">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stBlackLab">
              <property name="text">
               <string>Black</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stGreenLab">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stRedLab">
              <property name="text">
               <string>Red</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="statsBox">
           <property name="title">
            <string>Now</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="stAll">
              <property name="minimumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stBlack">
              <property name="minimumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stGreen">
              <property name="minimumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stRed">
              <property name="minimumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>34</width>
                <height>17</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
